core/ingest/src/main/java/mil/nga/giat/geowave/core/ingest/local/AbstractLocalFileDriver.java

```diff
@@ -66,29 +100,144 @@ protected boolean checkIndexesAgainstProvider(
 		return valid;
 	}
 
+	public void setURLStreamHandleFactory()
```

Comment:
"Handler?" - also do we know why we have this additional logic for setting s3 url handler but not for hdfs?

Reply:
oh, and another minor thing, can we just make this private